Error using discretize for Neumann and Robin boundary conditions, but not Dirichlet

Running examples from the heat equation [tutorial](http://methodoflines.sciml.ai/dev/tutorials/heat/) and my own problems, I get

`prob = discretize(pdesys,discretization)`

`ERROR: AssertionError: Number of offside points should not exceed the primary wind points`

using Neumann and Robin BCs, but it succeeds using Dirichlet.

(also a small note: the PDE system specification line in these docs should update from
`pdesys = PDESystem(eq,bcs,domains,[t,x],[u(t,x)])`
to
 `@named pdesys = PDESystem(eq,bcs,domains,[t,x],[u(t,x)]`)

--- a/docs/src/tutorials/heat.md
+++ b/docs/src/tutorials/heat.md
@@ -26,7 +26,7 @@ domains = [t ∈ Interval(0.0,1.0),
            x ∈ Interval(0.0,1.0)]
 
 # PDE system
-pdesys = PDESystem(eq,bcs,domains,[t,x],[u(t,x)])
+@named pdesys = PDESystem(eq,bcs,domains,[t,x],[u(t,x)])
 
 # Method of lines discretization
 dx = 0.1
@@ -79,7 +79,7 @@ domains = [t ∈ Interval(0.0,1.0),
         x ∈ Interval(0.0,1.0)]
 
 # PDE system
-pdesys = PDESystem(eq,bcs,domains,[t,x],[u(t,x)])
+@named pdesys = PDESystem(eq,bcs,domains,[t,x],[u(t,x)])
 
 # Method of lines discretization
 # Need a small dx here for accuracy
@@ -134,7 +134,7 @@ domains = [t ∈ Interval(0.0,1.0),
         x ∈ Interval(-1.0,1.0)]
 
 # PDE system
-pdesys = PDESystem(eq,bcs,domains,[t,x],[u(t,x)])
+@named pdesys = PDESystem(eq,bcs,domains,[t,x],[u(t,x)])
 
 # Method of lines discretization
 # Need a small dx here for accuracy

